pkg/images/integration/set.go

```diff
@@ -24,7 +24,10 @@ type Set interface {
 
 // NewSet returns a new Set instance.
 func NewSet(reporter integrationhealth.Reporter) Set {
-	registryFactory := registries.NewFactory(registries.FactoryOptions{})
+	registryFactory := registries.NewFactory(registries.FactoryOptions{
+		CreatorFuncsWithoutRepoList: registries.AllCreatorFuncsWithoutRepoList,
```

Comment:
why not just default to this similar to `CreatorFuncs`?

Reply:
I was being cautious to not impact other usages of the factory (such as those in the Sensor scanning workflows) that do not need special handling for repo list vs. not.